src/routes/search/components/SearchModal.js

```diff
@@ -19,20 +20,23 @@ const Wrapper = styled.View`
 
 type CategoryListItemType = {| model: CategoryModel, subCategories: Array<CategoryModel> |}
 
-type PropsType = {|
+export type PropsType = {|
   categories: CategoriesMapModel | null,
-  navigateToCategory: (cityCode: string, language: string, path: string) => void,
-  theme: ThemeType,
-  language: string,
-  cityCode: string,
-  closeModal: () => void
+  navigateToCategory: (cityCode: string, language: string, path: string) => StoreActionType,
+  theme: any,
+  language: string | null,
+  cityCode: string | null,
+  closeModal: () => void,
+  navigation: NavigationScreenProp<*>
 |}
 
 type StateType = {|
   query: string
 |}
 
 class SearchModal extends React.Component<PropsType, StateType> {
+  static navigationOptions = {}
```

Comment:
Why is this required?

Reply:
Just added it for consistency

Reply:
Consistency with what? Is this common?

Reply:
I mean e.g. in the `Dashboard` component you removed the `navigationOptions` completely.